mobile-banking-service/src/test/java/org/xpdojo/bank/cdc/mobile/pact/MobileConsumerAccountSummaryPactTest.java

```diff
@@ -18,68 +16,39 @@
 import org.xpdojo.bank.cdc.mobile.domain.Account;
 
 import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @ExtendWith(PactConsumerTestExt.class)
 @PactTestFor(providerName = "account_provider")
 public class MobileConsumerAccountSummaryPactTest {
 
-    private static final Logger LOG = LoggerFactory.getLogger(MobileConsumerAccountSummaryPactTest.class);
+	private static final Logger LOG = LoggerFactory.getLogger(MobileConsumerAccountSummaryPactTest.class);
 
-    @Pact(provider = "account_provider", consumer = "mobile_consumer")
-    public RequestResponsePact configureMockServer(PactDslWithProvider builder) {
-        return builder
-                .uponReceiving("Request for all accounts")
-                .path("/accounts/30002468/balance")
-                .method("GET")
-                .willRespondWith()
-                .status(200)
-                .headers(expectedHeaders())
-                .body(expectedAccountsBody())
-                .toPact();
-    }
+	@Pact(provider = "account_provider", consumer = "mobile_consumer")
```

Comment:
The Pact annotations are BS. Is there a way to do this programmatically within the test.

Annotations = bad news (generally) 🔥